### DISCUSSION TITLE
建议添加检测连接中断提示信息

### DISCUSSION THREAD

**Discussion by Climit** [lollipopkit/flutter_server_box#125] [Ideas]
目前命令行中执行reboot后就处于不可操作状态，虽然知道这是为什么，但是还是建议在下方或其他地方进行连接中断的提示信息，这样更加直观一点。添加提示是考虑到使用ServerBox的用户也有可能是刚入门Linux或其他用户，另外偶见执行exit后不返回主界面的情况。

执行reboot后返回主界面依旧显示系统信息如（开机时间、温度、网络、存储、CPU、内存）等信息，造成还在运行的假象。

包管理、docker、进程加载信息的速度过慢，不知道是我系统或网络的问题还是什么，另外在加载包管理的时候我之前与系统更新对比了一下，系统中会有百分百显示，建议app也增加相应显示，这样不会造成app卡及网络卡的假象。

**Comment by diybl** [comment]
似乎是dartssh2的bug

**Comment by lollipopkit** [comment]
已实现
